src/components/features/AddressInfos/AddressInfos.tsx

```diff
@@ -34,8 +36,11 @@ const AddressInfos: React.FC<Props> = ({
   assetBadgeOptions,
   showAddress = true,
 }) => {
-  const web3 = useWeb3();
   const { network } = useContext(NetworkContext);
+  const { data: bytecode, isLoading: isBytecodeLoading } = useBytecode({
+    address: isAddress(address) ? (address as Address) : undefined,
```

Comment:
```suggestion
    address: getAddress(address),
```

Reply:
I get the following error
<img width="1011" height="637" alt="image" src="https://github.com/user-attachments/assets/161c2f57-c7bb-4286-9c2a-b7a624cedd61" />

Refactored to this:

```ts
    address: isAddress(address) ? getAddress(address) : undefined,
```